backend/dep/wire.go

```diff
@@ -223,6 +225,7 @@ func InjectRoutingService(
 	dataDogAPIKey provider.DataDogAPIKey,
 	segmentAPIKey provider.SegmentAPIKey,
 	ipStackAPIKey provider.IPStackAPIKey,
+	searchTimeout time.Duration,
```

Comment:
I think it should be `provider.SearchTimeoutDuration`. i think if there were multiple `time.Duration` arguments then wire will fail to build as it will not know which time.Duration to use.

Reply:
@kapillamba4 Great comment! Wow, you left the exact comment I was trying to give!

Reply:
I think `provider.SearchTimeout` sounds better. The type itself is already duration.